site/en/docs/android/custom-tabs/guide-interactivity/index.md

```diff
@@ -0,0 +1,197 @@
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:gravity="center">
+
+    <Button xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/ct_toolbar_next"
+        android:layout_width="wrap_content"
+        android:layout_height="48dp"
+        android:layout_margin="8dp"
+        android:padding="8dp"
+        android:paddingStart="16dp"
+        android:paddingEnd="16dp"
+        android:text="Next" />
+
+    <Button xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/ct_toolbar_previous"
+        android:layout_width="wrap_content"
+        android:layout_height="48dp"
+        android:layout_margin="8dp"
+        android:padding="8dp"
+        android:text="Previous" />
+</LinearLayout>
+```
+
+{% Aside 'gotchas' %}
+The Custom Tab color scheme does not apply here, you need to manually style the toolbar. If your app provides a light and a dark scheme, you need to configure two different toolbars, one for each color scheme.
+{% endAside %}
+
+The next step is to register a [BroadcastReceiver](https://developer.android.com/guide/components/broadcasts#receiving-broadcasts), which handles toolbar interactions, in the AndroidManifest.xml file:
+
+```xml
+<application …>
+  <receiver android:name=".CustomTabBottomToolbarBroadcastReceiver" />
+</application>
+```
+
+Then implement the BroadcastReceiver, which will handle all interactions with the bottom toolbar:
+
+```java
+public class BottomToolbarBroadcastReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        String url = intent.getDataString();
+        int remoteViewId = intent.getIntExtra(EXTRA_REMOTEVIEWS_CLICKED_ID, -1);
+        if (remoteViewId == R.id.ct_toolbar_previous) {
+          // handle previous
+        } else if (remoteViewId == R.id.ct_toolbar_next) {
+          // handle next
+        }
+    }
+}
+```
+
+Finally, register the toolbar:
+
+```java
+// Create the pending intent
+Intent actionIntent = new Intent(
+       this.getApplicationContext(), BottomToolbarBroadcastReceiver.class);
+
+PendingIntent pendingIntent =
+       PendingIntent.getBroadcast(getApplicationContext(), 0 /* request code */, actionIntent, PendingIntent.FLAG_MUTABLE);           
+
+// Pass the toolbar layout to the RemoteViews instance
+RemoteViews secondaryToolbarViews = new RemoteViews(getPackageName(), R.layout.custom_tab_toolbar);
+// All toolbar buttons
+int[] clickableIds = {R.id.ct_toolbar_next, R.id.ct_toolbar_previous};
+
+// Register the bottom toolbar when creating a new custom tab intent
+CustomTabsIntent intent = new CustomTabsIntent.Builder()
+    .setSecondaryToolbarViews(secondaryToolbarViews, clickableIds, toolbarPendingIntent)
+    .build();
+```
+
+
+Next up: [Learn how to speed up loading web content in a Custom Tab](/docs/android/custom-tabs/warmup-prefetch/).
```

Comment:
is the link corect? I would expect it to be guide-warmup-prefetch from the filenames